css/styles.css

```diff
@@ -0,0 +1,243 @@
+*{
+    background-color: #ECEEF1;
+}
+
+/* tags formats */
```

Comment:
Remove all comments from your CSS and HTML files.

Reply:
Dear elmejdki,
Thank You for reviewing the code and suggestions.
We've intentionally added the comments for making it easy to navigate.
I'm just curious to know that why should we remove that?
Thanks!